packages/terra-badge/tests/jest/__snapshots__/Badge.test.jsx.snap

```diff
@@ -1,20 +1,20 @@
 exports[`test should render a badge component in the order, icon and text with medium size 1`] = `
 <span
-  className="terra-Badge terra-Badge--medium terra-Badge--default">
+  className="badge medium default">
   <img
     alt="Test icon" />
   <span
-    className="terra-Badge-text">
+    className="text">
```

Comment:
We can change it, but its worth noting that is localized to the component so that although the test shows `text`, the actual HTML output will either be a hash or will be a source path location of the selctor (in dev).